package.json

```diff
@@ -19,17 +17,26 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.0",
     "react-toastify": "^5.4.1",
-    "typescript": "^3.7.4"
+    "smcloudstore": "^0.2.1"
```

Comment:
What about typescript? @types?

Reply:
@types are need only at dev mode and build mode ryt? when in production when the typescript get compiled to js do we need it?. did this so that we can optimise the deployment with npm prune --production.

Reply:
But isn't it the main `package.json` of the project?